project_app/components/Footer.js

```diff
@@ -7,15 +7,18 @@ import {
   ProductContainer,
 } from "./styled";
 import Image from "next/image";
+import Link from "next/link";
 
 export default function Footer() {
   const titles = ["About Us", "FAQ", "Contact Us", "Give Feedback"];
 
   return (
     <Box>
       <ProductContainer>
-        {titles.map((title) => (
-          <ul key={title}>{title}</ul>
+        {titles.map((title, key) => (
+          <Link href="/About" key={key}>
+            <ul>{title}</ul>
```

Comment:
they gotta have their own links, not all the links point to /about

Reply:
pointed to separate pages